azure-pipelines.yml

```diff
@@ -32,9 +32,12 @@ steps:
       source $HOME/miniconda3/bin/activate
       echo Activate environment
       call conda activate test_
+      echo Install support packages
+      pip install pyserial
       echo Install project
       pip install git+https://github.com/pyvisa/pyvisa.git#egg=pyvisa
       pip install -e .
```

Comment:
You could do `pip install -e .[serial]` instead with a comment explining why we install this extra.

Reply:
If you would prefer it that way, I will change it.